#goal definition
string cocktail
---
#result definition
bool success
---
#feedback
string task_state
